www/src/utils/i18n.js

```diff
@@ -1,19 +1,59 @@
+const langs = require("../../i18n.json")
 const defaultLang = "en"
 
+const langCodes = langs.map(lang => lang.code)
+
+const localizedSections = ["tutorial"]
+
 function isDefaultLang(locale) {
   return locale === defaultLang
 }
 
+/**
+ * Get the path prefixed with the locale
+ * @param {*} locale the locale to prefix with
+ * @param {*} path the path to prefix
+ */
 function localizedPath(locale, path) {
   const isIndex = path === `/`
 
-  // TODO generalize this to other paths
-  const isLocalized = !isDefaultLang(locale) && path.startsWith("/tutorial/")
-  // If it's the default language, don't do anything
+  // Our default language isn't prefixed for back-compat
+  if (isDefaultLang(locale)) {
+    return path
+  }
+
+  const [, base] = path.split("/")
+
+  // If for whatever reason we receive an already localized path
+  // (e.g. if the path was made with location.pathname)
+  // just return it as-is.
+  if (langCodes.includes(base)) {
```

Comment:
Inspired to make this change because of docs-table-of-contents.js, which determines its path using location.pathname. Though I guess it's covered under the condition below, it's good to be explicit to prevent future bugs.